plugins/experimental/txn_box/plugin/include/txn_box/accl_util.h

```diff
@@ -65,7 +65,7 @@ template <typename Key, typename Value> class StringTree
     using self_type = Node;
     using ptr_type  = self_type *;
 
-    Node(Key k, Value v, int32_t r = -1) : key{k}, value{v}, rank{r}
+    Node(Key k, Value v, int32_t r = -1) : key{std::move(k)}, value{std::move(v)}, rank{r}
```

Comment:
is it guaranteed or given that when creating a Node object that `k` and `v` will not be used anymore since its contents have now been moved instead of copied?